Makefile

```diff
@@ -14,7 +14,8 @@ $(APPS):
 clone: $(APPS)
 
 down:
```

Comment:
I think it'd make sense to rename this to kill or something that represents the intent